Jobs/PrGate.yml

```diff
@@ -68,64 +64,94 @@ parameters:
   displayName: Virtual Machine Image (e.g. windows-latest)
   type: string
   default: 'windows-latest'
+- name: extra_install_step
+  displayName: Extra Install Steps
+  type: stepList
+  default: []
+- name: pool_name
+  displayName: Variable name that hosts pool name to be used for self-hosted agents
+  type: string
+  default: ''
+- name: os_type
+  displayName: OS type on the self-hosted agent pools
+  type: string
+  values:
+  - Windows_NT
+  - Linux
+  default: Windows_NT
+- name: build_matrix
+  type: object
+  # Each element in the matrix will be used in a separate job
+  # Required fields:
+  # Pkgs:           Specify what packages in the target repo are to be build for this job
+  # Targets:        Specify which targets are to be built in this job, i.e. DEBUG,
+  #                 RELEASE, NO-TARGET, NOOPT. The acceptable values depend on the
+  #                 Pkgs to be built.
+  # ArchList:       List of architectures to be supported by the packages (e.g. IA32, X64)
+  # SelfHostAgent:  A boolean indicating whether this job should be run on the selfhosted

```

Comment:
Is it possible to make SelfHostAgent conditional? If this value is not prevent then eq(item.Value.SelfHostAgent, true)) would be false still right? or just it fail?

Reply:
Agreed, this is updated.